phpBB/includes/functions_privmsgs.php

```diff
@@ -2040,7 +2040,7 @@ function message_history($msg_id, $user_id, $message_row, $folder, $in_post_mode
 	while ($row = $db->sql_fetchrow($result));
 	$db->sql_freeresult($result);
 
-	if (count($rowset) == 1 && !$in_post_mode)
+	if (count($rowset) == 0 && !$in_post_mode)
```

Comment:
`if (!$rowset && !$in_post_mode)` may be?

Reply:
Ah yes ;)